specs/trust-store-trust-policy.md

```diff
@@ -368,17 +368,17 @@ There are two types of CRLs (per RFC 3280), Base CRLs and Delta CRls.
   Delta CRLs are signed by the certificate issuing CAs.
 - **Indirect CRLs:** Special Case in which BaseCRLs and Delta CRLs are not signed by the certificate issuing CAs, instead they are signed with a completely different certificate.
 
-Notary v2 MUST support BaseCRLs and Delta CRLs.
-Notary v2 MAY support Indirect CRLs.
-Notary v2 supports only HTTP CRL URLs.
+Notary Project specification implementation MUST support BaseCRLs and Delta CRLs.
+Notary Project specification implementation MAY support Indirect CRLs.
+Notary Project specification implementation supports only HTTP CRL URLs.
 
 Implementations MAY add support for caching CRLs and OCSP response to improve availability, latency and avoid network overhead.
 
 ##### CRL Download
 
 CRL download location (URL) can be obtained from the certificate's CRL Distribution Point (CDP) extension.
 If the certificate contains multiple CDP locations then each location download is attempted in sequential order.
-For each CDP location, Notary V2 will try to download the CRL for the default threshold of 10 seconds.
+For each CDP location, Notary Project specification implementation will try to download the CRL for the default threshold of 10 seconds.
```

Comment:
For each CDP location, the Notary Project specification implementation

Reply:
update it to `Implementations of the Notary Project specification`